### PR TITLE
Fix: webpack polyfill error

### PR DESCRIPTION
**Pull request by timkim**
For some reason webpack decided to get angry about not having tty and os

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/gatsby-theme-aio/package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "builtin-status-codes": "^3.0.0",
     "classnames": "^2.2.6",
     "core-js": "^3.16.3",
+    "dependency": "^0.0.1",
```
Comment (icaraps) [inline packages/gatsby-theme-aio/package.json:83]: Mhh looks weird why is it needed ?

Comment (icaraps) [reply packages/gatsby-theme-aio/package.json:83]: @timkim looks like this is failing the tests

Comment (timkim) [reply packages/gatsby-theme-aio/package.json]: Oh I accidently added `dependency` and my work machine was all configured for artifactory. The latest commit should fix it now.

File: yarn.lock [hunk 2]
```diff
@@ -36,18 +36,37 @@
   dependencies:
     "@algolia/cache-common" "4.10.4"
 
+"@algolia/cache-browser-local-storage@4.11.0":
```
Comment (icaraps) [inline yarn.lock:39]: @timkim I reverted the lock file update because it caused some weird issues ..

**Pull request merged**